docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+name: ultrasinger
```

Comment:
Do we need this name here? Isnt container_name enough?

Reply:
i have added this to the two docker-compose.yml files

Reply:
i did need to add a .env file to make this work.

Reply:
i removed it.